x-pack/legacy/plugins/integrations_manager/server/integrations/get.ts

```diff
@@ -10,10 +10,14 @@ import { InstallationAttributes } from '../../common/types';
 import * as Registry from '../registry';
 import { createInstallableFrom } from './index';
 
+function nameAsTitle(name: string) {
+  return name.charAt(0).toUpperCase() + name.substr(1).toLowerCase();
```

Comment:
Have been thinking about this again. What happens for `IIS` Integration here?

Reply:
I think it's a "requirement" we could already enforce in the packages. So if a title is not set, it's not a valid package and can't be added to the registry.